make/dpcpp.inc

```diff
@@ -72,5 +72,5 @@ ifeq ($(device_type), FPGA_EMULATOR)
     DEVICE_COMPILE_FLAGS += $(KEY)DONEDPL_FPGA_EMULATOR
 endif
 ifeq ($(ranges_api), 1)
-    CPLUS_FLAGS += $(KEY)D_ONEDPL_USE_RANGES=1
+    CPLUS_FLAGS += $(KEY)D_ENABLE_RANGES_TESTING=1
```

Comment:
I suggest removing this line as we infer the macro in `pstl_test_config.h`. Let's improve the conditions within `pstl_test_config.h` if we need to pass the macro.

If you decide to remove this line, then remove `#if !defined(_ENABLE_RANGES_TESTING)` line in `pstl_test_config.h`

Reply:
I know about definition of `_ENABLE_RANGES_TESTING` in `pstl_test_config.h`.
BTW, that definition is wrapped into `#if !defined(_ENABLE_RANGES_TESTING)`.
Sometime, for experiments and "debug"  I (as a developer) need to control the "ranges API enabling" directly from "make"  command line, by the option `ranges_api=1`. So, I has kept this option on purpose.